src/Writers/HTMLWriter.jl

```diff
@@ -977,6 +1073,41 @@ function asset_links(src::AbstractString, assets::Vector{HTMLAsset})
     return links
 end
 
+function plugin_assets(ctx::HTMLContext)
+    for (type, plugin) in ctx.doc.plugins
+        assets = plugin_html_assets(type)
+        if !isempty(assets)
+            # no path checking is done here as the plugin authors have to
+            # take care themselves that valid paths are provided (as the path
+            # completion for the relative source files in the asset()
+            # function will only complete for the docs/src directory)!
+            # so @blugin_author: no fancy asset() autocompletion for you
```

Comment:
```suggestion
            # completion for the relative source files in the asset()
            # function will only complete for the docs/src directory)!
            # so @plugin_author: no fancy asset() autocompletion for you
```

Also, is the autocompletion comment still valid?

Reply:
I think it is still valid as the file paths paths provided by **plugin authors** should be absolute paths as there is no way to derive the source file position from a relative path afterwards.
The only cases where relative paths are allowed are by `Documenter.jl` users as we assume relative paths are pointing to files in `<dev_dir>/docs/src` (although there are no checks in place for that, or?).